docs/2.6.5/docs/daml/reference/choices.rst

```diff
@@ -56,7 +56,6 @@ Choice Name
 
 - The name of the choice. Must begin with a capital letter.
 - If you're using choice-first, preface with ``choice``. Otherwise, this isn't needed.
-- Must be unique in your project. Choices in different templates can't have the same name.
```

Comment:
I don't think this has ever been true, at least not since I joined

Reply:
AFAIK It's true within a file/module, but not across a project.

Reply:
thanks, I've updated to reflect this